pkg/reconciler/knativeeventing/knativeeventing.go

```diff
@@ -106,7 +106,7 @@ func (r *Reconciler) ReconcileKind(ctx context.Context, ke *v1alpha1.KnativeEven
 		common.CheckDeployments,
 		common.DeleteObsoleteResources(ctx, ke, r.installed),
 	}
-	manifest := r.manifest.Append()
+	manifest := r.manifest.Append(r.extension.Manifests()...)
```

Comment:
why is this different than here:

https://github.com/knative/operator/pull/468/files#diff-fc7141af4d915a54d757eac679ca6294d9d25d2493f12c8ad910afdb6678b906R104-R143

Reply:
Something is wrong with the cherry picking. Let me check.

Reply:
@matzew now its ok. Sorry for the messed up patch.